README.md

```
@@ -91,6 +91,10 @@ To delete all resources in a Terraform state file:
 
     terradozer [flags] <path/to/terraform.tfstate|s3://bucket/key>
 
+To delete all resources in every `.tfstate` file under a local directory or S3 prefix:
+
+    terradozer -recursive [flags] <directory|s3://bucket/prefix/>
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)